### PR TITLE
Clean up docs

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

**Comment by keradus** [comment]
why not 1.11 ?

**Comment by SpacePossum** [comment]
let me try again ;)

File: Symfony/CS/Tokenizer/Tokens.php [hunk 1]
```diff
@@ -1024,7 +1024,7 @@ public function isAnyTokenKindsFound(array $tokenKinds)
     /**
      * Check if token kind given as argument is found.
      *
-     * @param array
+     * @param int|string $tokenKind
```
Comment (SpacePossum) [inline Symfony/CS/Tokenizer/Tokens.php]: this fix must be done on master, the method it is not available on 1.x

Comment (keradus) [inline Symfony/CS/Tokenizer/Tokens.php]: what can be fix on 1.x line it should be fixed there. all the rest on master ;)
